### PR TITLE
tests: remove --module-rules-options instances

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
tests

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**
removed `--module-rules-options` instances as it has been excluded for cli from core schema. 

**Does this PR introduce a breaking change?**
NO

**Other information**
https://github.com/webpack/webpack/pull/11171

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@snitin315 problem on npm side https://status.npmjs.org/, don't worry

File: test/core-flags/module-flags.test.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 const { run, hyphenToUpperCase } = require('../utils/test-utils');
 const { flagsFromCore } = require('../../packages/webpack-cli/lib/utils/cli-flags');
-//--module-rules-options will be excluded for cli.
-const moduleFlags = flagsFromCore.filter(({ name }) => name.startsWith('module-') && name !== 'module-rules-options');
+
+const moduleFlags = flagsFromCore.filter(({ name }) => name.startsWith('module-'));
```
Comment (evilebottnawi) [inline test/core-flags/module-flags.test.js:6]: /cc @snitin315 So we still don't have tests on `module-` flags?

Comment (snitin315) [reply test/core-flags/module-flags.test.js:6]: @evilebottnawi  We have, This file contains exclusively tests for all `module-*` flags, We are extracting all the `module-*` flags here and then looping through them. 
https://github.com/webpack/webpack-cli/blob/4fe80450aa4c29f3603b5cecea7d2a0f2db59658/test/core-flags/module-flags.test.js#L8-L9

Comment (evilebottnawi) [reply test/core-flags/module-flags.test.js:6]: Thanks for clarify!

**Pull request merged**